docs/contributing/plugins/README.md

```diff
@@ -28,26 +29,62 @@ basicPluginBuilder(
     # You can add more names after the inner column if multiple plugins use
     # this layout
     ('My plugin name',),
-    # A list of parameter indexes
-    [45, 46, 35, 37, 218, 219, 220, 221],
-    # A color to represent the parameters (this can also be a list)
-    Color.fromInteger(0x206cc8)
+    # A list of parameter indexes - these will map left to write to the 
+    # relevant faders on your device, so in this example tuning will be
+    # fader 1, Waveform will be fader 2, Cutoff will be Fader 3, Resonance
+    # will be Fader 4 and so on.
+    [
+    0,  # Tuning
+    1,  # Waveform
+    2,  # Cutoff
+    4,  # Resonance
+    5,  # Envelope Modulation
+    6,  # Decay
+    7,  # Accent
+    8,  # Volume
```

Comment:
It could be a good idea to use different numbers for these, as currently, it could be interpreted as being the fader numbers. Maybe use these values:

```py
[
    45,  # Tuning
    46,  # Waveform
    35,  # Cutoff
    37,  # Resonance
    218,  # Envelope Modulation
    219,  # Decay
    220,  # Accent
    221,  # Volume
]
```

It might also be a good idea to mention that these indexes can be found using the [`fl_param_checker`](https://github.com/MiguelGuthridge/fl_param_checker) tool that is included with the script. I haven't written especially good documentation for that yet, but I'll get to it eventually.